# Because of how coverage is run, the current directory is the first in
# sys.path. This is a problem for the tests, because they may import a module of
# the same name as a module in the current directory.
diff --git a/coverage/cmdline.py b/coverage/cmdline.py
index dbf66e0a..780505ac 100644
--- a/coverage/cmdline.py
+++ b/coverage/cmdline.py
@@ -937,6 +937,7 @@ def main(argv=None):
     This is installed as the script entry point.
 
     """
+    sys.path.append(sys.path.pop(0))
     if argv is None:
         argv = sys.argv[1:]
     try:
